doc/en/getting-started.rst

```diff
@@ -185,6 +185,21 @@ You can find out what kind of builtin :ref:`fixtures` exist by typing::
 
     pytest --fixtures   # shows builtin and custom fixtures
 
+Contributing tests to an existing project
```

Comment:
> I disagree on the 'people being used to virtualenvs' point, many users coming to a getting started section will not be used to any of the basic concepts of Python or unit testing. 

You may be right, I'm probably biased on this. But perhaps this should be moved somewhere in [goodpractices](http://doc.pytest.org/en/latest/goodpractices.html) then? `virtualenv`, `tox`, etc are all mentioned there.